index.ts

```diff
@@ -0,0 +1,727 @@
+import axios from 'axios';
+import TelegramBot, { Message } from 'node-telegram-bot-api';
+import * as dotenv from 'dotenv';
+import { TVLResponse, VolumeResponse, DailyVolumeResponse } from './src/types';
+import { generateColorPalette } from './src/helper';
+
+
+dotenv.config();
+
+// Replace the value below with the Telegram token you receive from @BotFather
+const token: string = process.env.TELEGRAM_BOT_TOKEN as string;
+
+const bot = new TelegramBot(token, {
+  polling: {
+    interval: 3000, // 3 seconds between polling requests
+    autoStart: true, 
+    params: {
+      timeout: 10 // Long polling timeout (in seconds)
+    }
+  }
+});
+
+// Set bot commands so they appear in the menu
+bot.setMyCommands([
+  { command: '/start', description: 'Start' },
+  { command: '/get_tvls', description: 'Get the current TVLs for Raindex' },
+  { command: '/get_volume', description: 'Get the total volume data for Raindex' },
+  { command: '/get_daily_volume', description: 'Get daily volume per chain for raindex' },
+  { command: '/get_daily_token_distribution', description: 'Get token distrubition' },
+  { command: '/get_most_traded_tokens', description: 'Get most traded tokens on raindex' },
+  { command: '/get_monthly_volume', description: 'Get monthly volumes for past 12 months.' },
```

Comment:
this is the only description with a period at the end?

Reply:
resolved